ocaml/typing/env.ml

```diff
@@ -4353,8 +4164,8 @@ let report_lookup_error _loc env ppf = function
       in
```

Comment:
I suspect we want to add a `Style.as_inline_code` in the `Functor_used_as_structure` case above (I can't add a comment there)